tutorials/editor/external_editor.rst

```diff
@@ -65,6 +67,9 @@ Some example **Exec Flags** for various editors include:
     For Emacs, you can call ``emacsclient`` instead of ``emacs`` if
     you use the server mode.
 
+    For Visual Studio, you will have to open the solution file ``.sln``
+    manually to get access to the IDE features. Additionally, it will not go to a specific line.
```

Comment:
Doesn't Visual Studio have a syntax such as a `:{line}` suffix in the filename to specify a line to go to?

Reply:
I refered to [devenv-command-line-switches](https://learn.microsoft.com/en-us/visualstudio/ide/reference/devenv-command-line-switches?view=vs-2022) which has the `/edit` command which accepts only a file.  
Goto line exists within `/command goto` kind of syntax, but it can not be chained with the initial `/edit`.  
I have not found any other possibility.